src/hex/algorithms.rs

```diff
@@ -211,4 +227,10 @@ mod tests {
             ]
         );
     }
+
+    #[test]
+    fn reflect() {
+        assert_eq!(axial!(1,1).reflect(None, Axes::Q), axial!(1, -2));
+        assert_eq!(axial!(1,1).reflect(Some(axial!(1,2)), Axes::Q), axial!(1, 3));
```

Comment:
Might be useful to have a test case where the value of `q` changes.

Reply:
testing: fixed in https://github.com/algodiva/Gridly/pull/8/commits/3a8c1621da359af7681d20a6f5ee2a8028b4104f

Reply:
New test cases look good to me.